app/include/zmk/hid.h

```diff
@@ -135,6 +135,38 @@ static const uint8_t zmk_hid_report_desc[] = {
     ZMK_HID_CONSUMER_NKRO_SIZE,
     HID_MI_INPUT,
     0x00,
+
+#ifdef CONFIG_ZMK_USB_REPORT_LEDS
+    /* LED */
+    0x85, 0x01,
+    /* REPORT_COUNT (5) */
+    HID_GI_REPORT_COUNT,
+    0x05,
+    /* REPORT_SIZE (1) */
+	  HID_GI_REPORT_SIZE,
```

Comment:
You should install clang-format (https://releases.llvm.org/download.html) and run your code through it. Otherwise the automated checks will complain that this isn't formatted correctly.

If you're using VS Code and clang-format is installed, Microsoft's C/C++ extension can format files with Shift+Alt+F or automatically on save by setting the `editor.formatOnSave` setting.